docs/source/markdown/podman-create.1.md

```diff
@@ -599,6 +599,15 @@ Whether to disable OOM Killer for the container or not.
 
 Tune the host's OOM preferences for containers (accepts -1000 to 1000)
 
+**--override-arch**=*ARCH*
+Override the machine's default architecture of the image to be pulled. For example, `arm`.
```

Comment:
```suggestion
Override the machine's default architecture of the image to be created. For example, `arm`.
```

Reply:
You are not creating an image, you are pulling it.